lib/formalism/form/coercion.rb

```diff
@@ -75,9 +75,14 @@ def to_float
 		end
 
 		def to_time
-			return if @value.nil?
-
-			Time.parse(@value)
+			case @value
+			when String
+				Time.parse @value
+			when Integer
+				Time.at @value
+			end
+		rescue ArgumentError => exception
+			raise exception unless exception.message.include? 'out of range'
```

Comment:
`raise exception` can be replaced with `raise` inside `rescue` block.

Reply:
I didn't know it, thanks! Changed.